server/src/db_proxy_server/business/UserModel.cpp

```diff
@@ -12,7 +12,7 @@
 #include "../DBPool.h"
 #include "../CachePool.h"
 #include "Common.h"
-#include "SyncCenter.h"
+#include "../SyncCenter.h"
```

Comment:
I did not have a thorough look of what `.sh` script did But it seems bump some `.h` files into this folder and now we put it into a shared folder?

Reply:
No, those build `.sh` scripts don't copy common headers, but there are duplicates in different directories (I guess they must be copied manually before committing to Git).

I removed as many duplicates as I can and tweaked the include directories in the CMakeLists.txt files to pass the compilation.